### PR TITLE
Add bg-BG localization

### PR DESCRIPTION
**Pull request by zlatkow1996**
Please review the localization for Bulgarian.

### CODE REVIEW

File: src/Notepads/Strings/bg-BG/Settings.resw [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+  </data>
+  <data name="TextAndEditorPage_SearchEngineSettings_Title.Text" xml:space="preserve">
+    <value>Търсачка по подразбиране</value>
+    <comment>TextAndEditorPage SearchEngineSettings Title display text.</comment>
+  </data>
+  <data name="TextAndEditorPage_DecodingSettings_AutoGuessRadioButton.Content" xml:space="preserve">
+    <value>Автоматично познаване кодиране (препоръчително)</value>
+    <comment>TextAndEditorPage DecodingSettings AutoGuessDecodingRadioButton content display text.</comment>
+  </data>
+  <data name="String1" xml:space="preserve">
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Settings.resw:376]: Can you remove this empty resource data field?

Comment (zlatkow1996) [reply src/Notepads/Strings/bg-BG/Settings.resw]: The empty resource data field is removed.

File: src/Notepads/Strings/bg-BG/Resources.resw [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+    <value>Настройки</value>
+    <comment>MainMenu: "Settings" button display text.</comment>
+  </data>
+  <data name="RevertAllChangesConfirmationDialog_CloseButtonText" xml:space="preserve">
+    <value>Отказ</value>
+    <comment>RevertAllChangesConfirmationDialog: CloseButtonText.</comment>
+  </data>
+  <data name="RevertAllChangesConfirmationDialog_Content" xml:space="preserve">
+    <value>Всички промени, включително текст, край на реда и кодиране, направени на "{0}",
+ ще бъдат възстановени!</value>
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Resources.resw:262]: Put it in one line.

File: src/Notepads/Strings/bg-BG/Resources.resw [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+    <value>Премести отдел тук</value>
+    <comment>App: DragAndDrop UIOverride Caption: "Move tab here" display text</comment>
+  </data>
+  <data name="App_DragAndDrop_UIOverride_Caption_OpenWithNotepads" xml:space="preserve">
+    <value>Отвори с Notepads</value>
+    <comment>App: DragAndDrop UIOverride Caption: "Open with Notepads" display text</comment>
+  </data>
+  <data name="App_ShadowWindowIndicator_Description" xml:space="preserve">
+    <value>Това е прозорец за сянка на Notepads.
+Снимката и настройките на сесията са деактивирани.</value>
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Resources.resw:443]: Same here.

File: src/Notepads/Strings/bg-BG/Settings.resw [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+    <value>СОФТУЕРЪТ СЕ ПРЕДОСТАВЯ "КАКТО Е", БЕЗ НИКАКВИ ГАРАНЦИИ, ИЗРИЧНИ ИЛИ ПОДРАЗБИРАЩИ СЕ, ВКЛЮЧИТЕЛНО , НО НЕ САМО, ГАРАНЦИИТЕ ЗА ПРОДАВАЕМОСТ, ГОДНОСТ ЗА ОПРЕДЕЛЕНА ЦЕЛ И НЕНАРУШЕНИЕ. АВТОРИТЕ ИЛИ НОСИТЕЛИТЕ НА АВТОРСКИ ПРАВА НЕ НОСЯТ НИКАКВА ОТГОВОРНОСТ ЗА ИСК, ВРЕДА ИЛИ ДРУГА ОТГОВОРНОСТ, ПРОИЗТИЧАЩИ ОТ ИЗВЪН ИЛИ ВЪВ ВРЪЗКА СЪС СОФТУЕРА ИЛИ С ИЗПОЛЗВАНЕТО ИЛИ ДРУГИ СДЕЛКИ В СОФТУЕРА.</value>
+    <comment>AboutPage Disclaimer Content display text. (The MIT License Disclaimer)</comment>
+  </data>
+  <data name="AboutPage_Disclaimer_Title.Text" xml:space="preserve">
+    <value>Отказ от право</value>
+    <comment>AboutPage Disclaimer Title display text.</comment>
+  </data>
+  <data name="AboutPage_NotepadsShortDescription.Text" xml:space="preserve">
+    <value>Безплатен и с отворен код текстов редактор, проектиран и изпълнен от Jackie (Jiaqi)
+ Liu</value>
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Settings.resw:134]: Same here.

File: src/Notepads/Strings/bg-BG/Settings.resw [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+    <value>Краен ред по подразбиране</value>
+    <comment>TextAndEditorPage LineEndingSettings Title display text.</comment>
+  </data>
+  <data name="TextAndEditorPage_TabKeySettings_DefaultRadioButton.Content" xml:space="preserve">
+    <value>По подразбиране (\ t)</value>
+    <comment>TextAndEditorPage TabKeySettings DefaultRadioButton content display text.</comment>
+  </data>
+  <data name="TextAndEditorPage_TabKeySettings_Description.Text" xml:space="preserve">
+    <value>Настройки за поведение на клавиша на раздела, важат само за нови раздели, 
+вмъкнати от потребителя.</value>
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Settings.resw:259]: Same here.

File: src/Notepads/Strings/bg-BG/Settings.resw [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+    <value>Маркирайте грешно написани думи</value>
+    <comment>TextAndEditorPage SpellingSettings HighlightMisspelledWordsToggleSwitch On display text.</comment>
+  </data>
+  <data name="TextAndEditorPage_SpellingSettings_Title.Text" xml:space="preserve">
+    <value>Правопис</value>
+    <comment>TextAndEditorPage SpellingSettings Title display text.</comment>
+  </data>
+  <data name="AdvancedPage_AlwaysOpenNewWindow_Description.Text" xml:space="preserve">
+    <value>След като бъде активиран, Notepads винаги ще отварят файл в нов прозорец, 
+вместо да създават нов раздел.</value>
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Settings.resw:328]: Same here.

File: src/Notepads/Strings/bg-BG/Settings.resw [hunk 7]
```diff
@@ -0,0 +1,376 @@
...
+    <value>Персонализирана търсачка</value>
+    <comment>TextAndEditorPage CustomSearchUrlRadioButton display text.</comment>
+  </data>
+  <data name="TextAndEditorPage_SearchEngineSettings_CustomSearchUrlRadioButton_CustomUrlErrorReport.Text" xml:space="preserve">
+    <value>* Въведете URL формат https://www.example.com/search?q={0}</value>
+    <comment>TextAndEditorPage CustomSearchUrl Textbox display text when entered text is not a valid format.</comment>
+  </data>
+  <data name="TextAndEditorPage_SearchEngineSettings_Description.Text" xml:space="preserve">
+    <value>Настройките за търсачката по подразбиране се избират, когато се извърши 
+търсене в мрежата.</value>
```
Comment (JasonStein) [inline src/Notepads/Strings/bg-BG/Settings.resw:365]: Same here.

**Pull request merged**